recipes/xorg/all/conanfile.py

```diff
@@ -42,22 +42,22 @@ def system_requirements(self):
             if tools.os_info.with_apt:
                 packages = ["xorg-dev", "libx11-xcb-dev", "libxcb-render0-dev", "libxcb-render-util0-dev", "libxcb-xkb-dev",
                             "libxcb-icccm4-dev", "libxcb-image0-dev", "libxcb-keysyms1-dev", "libxcb-randr0-dev", "libxcb-shape0-dev",
-                            "libxcb-sync-dev", "libxcb-xfixes0-dev", "libxcb-xinerama0-dev", "xkb-data"]
+                            "libxcb-sync-dev", "libxcb-xfixes0-dev", "libxcb-xinerama0-dev", "xkb-data", "libxcb-util-dev"]
```

Comment:
```suggestion
                            "libxcb-sync-dev", "libxcb-xfixes0-dev", "libxcb-xinerama0-dev", "xkb-data", "libxcb0-util-dev" if tools.os_info.linux_distro == "ubuntu" and tools.os_info.os_version <= "14" else "libxcb-util-dev"]
```
gcc 4.9 docker image is based on trusty, which does not have `libxcb-util-dev` but `libxcb0-util-dev` while more recent distributions don't have `libxcb0-util-dev`